### PR TITLE
Glossary: Slashing section added

### PR DESCRIPTION
**Pull request by har00ga**
Ref. #948 

This PR includes an initial draft for the glossary's slashing section and explains both forms of penalties. 

I wasn't sure if I should provide a couple sentences to describe each of the Storage Fault types, if that's preferred let me know.

### CODE REVIEW

File: src/glossary/_index.md [hunk 1]
```diff
@@ -228,6 +228,24 @@ A sector is a contiguous array of bytes that a miner puts together, seals, and p
 
 ## Slashing
 
+It is important to understand that Filecoin implements two kinds of slashing: **Storage Fault Slashing** and **Consensus Fault Slashing**. Below are definitions of both types.
```
Comment (mishmosh) [inline src/glossary/_index.md:231]:
```suggestion
Filecoin implements two kinds of slashing: **Storage Fault Slashing** and **Consensus Fault Slashing**. Below are definitions of both types.
```

Comment (mishmosh) [reply src/glossary/_index.md:231]: Probably hundreds of these very important details so suggesting this simplification. Not a blocking comment.

File: src/glossary/_index.md [hunk 2]
```diff
@@ -228,6 +228,24 @@ A sector is a contiguous array of bytes that a miner puts together, seals, and p
...
+Filecoin implements two kinds of slashing: **Storage Fault Slashing** and **Consensus Fault Slashing**. Below are definitions of both types.
+
+### Storage fault slashing
+
+Storage Fault Slashing is a term that is used to encompass a broader set of penalties, including (but not limited to) Fault Fees, Sector Penalties, and Termination Fees. These penalties are to be paid by miners if they fail to provide sector reliability or decide to voluntarily exit the network. 
+
+* **Fault Fee (FF):** A penalty that a miner incurs for each day a miner's sector is offline.
+* **Sector Penalty (SP):**  A penalty that a miner incurs for a faulted sector that was not declared faulted ahead of time. 
+    * The sector will pay FF after incurring an SP when the fault is detected.
+* **Termination Fee (TP):** A penalty that a miner incurs when a sector is voluntarily terminated and exited from the network.
```
Comment (zixuanzh) [inline src/glossary/_index.md:240]: nit: voluntarily or involuntarily

File: src/glossary/_index.md [hunk 3]
```diff
@@ -228,6 +228,24 @@ A sector is a contiguous array of bytes that a miner puts together, seals, and p
...
 ## Slashing
 
+Filecoin implements two kinds of slashing: **Storage Fault Slashing** and **Consensus Fault Slashing**. Below are definitions of both types.
+
+### Storage fault slashing
+
+Storage Fault Slashing is a term that is used to encompass a broader set of penalties, including (but not limited to) Fault Fees, Sector Penalties, and Termination Fees. These penalties are to be paid by miners if they fail to provide sector reliability or decide to voluntarily exit the network. 
+
+* **Fault Fee (FF):** A penalty that a miner incurs for each day a miner's sector is offline.
+* **Sector Penalty (SP):**  A penalty that a miner incurs for a faulted sector that was not declared faulted ahead of time.
```
Comment (zixuanzh) [inline src/glossary/_index.md:238]: nit: ... ahead of time --> ... ahead of a WindowPoSt check

**Pull request merged**